src/requirements/README.md

```diff
@@ -32,10 +33,11 @@ The requirements data and its interpreting algorithm is uniquely developed by an
 |  name | full name of the requirement | any string  |
 | description  | description of the requirement  | string copied from source  |
 | source | the soure with more information on the requirement | a URL string |
-| search | the command used to determine whether courses satisfy the requirement | (special search: all, all-elidgible, code) or any attribute in course object |
-| includes | the query that uses the search command  | a string that is an value in course data (e.g CS 2110, MQR-AS) |
+| search | (optional if fulfilledBy exists) the command(s) used to determine whether courses satisfy the requirement | (special search: all, all-elidgible, code) or any attribute in course object |
+| includes | the query that uses the search command to find courses that fulfills requirement | a string that is an value in course data (e.g CS 2110, MQR-AS) |
+| excludes | (optional) the query that uses the search command to find courses that does not fulfill requirement | a string that is an value in course data (e.g CS 2110, MQR-AS) |
```

Comment:
Excludes works for me for English! Is this the only major it applies to right now?

Reply:
Exclude simplifies defining requirements. It's not a new feature entirely. For example includes INFO 1*** and excludes INFO 18** is the same as includes INFO 10**, INFO 11**, INFO 12**, INFO 13** ... INFO 19**. INFO is a major that could've been more simplified with excludes should it have been implemented when we first inputted data for it. I'm sure there are others like it that we will revamp when we need to revise it